src/main/java/spireMapOverhaul/zones/gremlinTown/relics/WarHorn.java

```diff
@@ -0,0 +1,37 @@
+package spireMapOverhaul.zones.gremlinTown.relics;
+
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import spireMapOverhaul.abstracts.AbstractSMORelic;
+import spireMapOverhaul.util.Wiz;
+import spireMapOverhaul.zones.gremlinTown.GremlinTown;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+import static spireMapOverhaul.util.Wiz.adp;
+import static spireMapOverhaul.util.Wiz.att;
+
+public class WarHorn extends AbstractSMORelic {
+    public static final String ID = makeID(WarHorn.class.getSimpleName());
+    private static final int STRENGTH_AMOUNT = 3;
+
+    public WarHorn() {
+        super(ID, GremlinTown.ID, RelicTier.SPECIAL, LandingSound.CLINK);
+    }
+
+    public void atBattleStart() {
```

Comment:
I guess you don't just check for the room because of the Mindbloom event?

Reply:
Correct.  This is consistent with Pantograph behavior, which heals you during said event.